.circleci/config.yml

```diff
@@ -7,22 +7,21 @@ executors:
   default:
     working_directory: ~/timetree-sdk-js/web-api
     docker:
-      - image: circleci/node:12.19.0
+      - image: cimg/node:14.15.1
 
 commands:
   build_npm:
     steps:
       - restore_cache:
           keys:
             - v1-dependencies-{{ arch }}-{{ checksum "../.node-version" }}-{{ checksum "yarn.lock" }}
-            - v1-dependencies-{{ arch }}
-            - v1-dependencies-
+            - v1-dependencies-{{ arch }}-{{ checksum "../.node-version" }}-
       - run:
           name: Yarn
-          command: yarn
+          command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn
```

Comment:
Fixed according to best practices.
https://circleci.com/docs/2.0/caching/#yarn-node